locale/hi/LC_MESSAGES/loris.po

```diff
@@ -396,6 +396,9 @@ msgstr "प्रगति पर"
 msgid "Complete"
 msgstr "पूर्ण"
 
+msgid "Instrument"
```

Comment:
Different languages have different numbers of plurals. This should be

```
msgid "Instrument"
msgid_plural "Instruments"
msgid[0] = "उपकरण"
msgid[1] = "उपकरण"
```
(or similar. I'm not sure if it's correct that singular/plural are the same here, I just copy/pasted.)

Reply:
and then use either dcngettext instead of dgettext or add a count: to the js t() call.